src/util/rule.js

```diff
@@ -0,0 +1,62 @@
+import Ticket from "../domain/Ticket.js";
+import WinningDetail from "../domain/WinningDetail.js";
+
+export const RULES = {
+  MIN_TICKET_NUMBER: 1,
+  MAX_TICKET_NUMBER: 45,
+  MIN_LOTTO_PURCHASE_PRICE: 1000,
+  MAX_LOTTO_PURCHASE_PRICE: 100000,
```

Comment:
4자리가 넘어가는 숫자를 사용할 때는 읽기 좋게 numeric seperator를 사용해보셔도 좋을거 같아요!
```suggestion
  MAX_LOTTO_PURCHASE_PRICE: 100_000,
```

Reply:
안 그래도 이 부분 쓰면서도 이상해서 찾고 있었는데, 알려주셔서 감사합니다!!